librz/bin/format/pyc/marshal.c

```diff
@@ -437,13 +437,34 @@ static pyc_object *get_unicode_object(RzBuffer *buffer) {
 	if (error) {
 		return NULL;
 	}
+	ut64 addr = rz_buf_tell(buffer);
 	ret = RZ_NEW0(pyc_object);
 	ret->type = TYPE_UNICODE;
 	ret->data = get_bytes(buffer, n);
 	if (!ret->data) {
 		RZ_FREE(ret);
 		return NULL;
 	}
+
+	if (n > 0) {
+		RzBinString *string = NULL;
+		string = RZ_NEW0(RzBinString);
+		if (!string) {
+			RZ_FREE(ret);
+			return NULL;
+		}
+		string->paddr = string->vaddr = addr;
```

Comment:
is there no virtual vs physical distinction in Pyc?

Reply:
as far as I understand no, there is no distinction